packages/vuetify/src/components/VSelect/VSelectList.ts

```diff
@@ -146,11 +146,12 @@ export default mixins(Colorable, Themeable).extend({
       const end = text.slice(index + searchInput.length)
       return { start, middle, end }
     },
-    genTile (
-      item: object,
-      disabled = null as null | boolean,
-      value = false
-    ): VNode | VNode[] | undefined {
+    genTile ({
+      item,
+      index,
+      disabled = null,
+      value = false,
+    }: { item: any, disabled?: null | boolean, value?: boolean, index: number }): VNode | VNode[] | undefined {
```

Comment:
This seems like it should be a specific type.

Reply:
Added :)